### PR TITLE
adds poor screenwriter's note

### PR DESCRIPTION
**Pull request by Coxswain-Navigator**
## About The Pull Request
Adds poor screenwriter's note

## Why It's Good For The Game
Is it?

## Changelog
:cl:
add: Added poor screenwriter's note
/:cl:

### CODE REVIEW

**Comment by TravisAngeI** [comment]
Minor misspell on the 'A' simplemob's examine I think, it says 'screepy' when I think it should be 'creepy'

File: code/modules/mob/living/simple_animal/abnormality/waw/screenwriter.dm [hunk 1]
```diff
@@ -0,0 +1,376 @@
...
+
+	//Manually Add/Log PE
+	src.datum_reference.stored_boxes += pe
+	SSlobotomy_corp.AdjustAvailableBoxes(pe)
+	var/user_job_title = "Unidentified Employee"
+	var/obj/item/card/id/W = user.get_idcard()
+	if(istype(W))
+		user_job_title = W.assignment
+	datum_reference.work_logs += "\[[worldtime2text()]\] [name]: [user_job_title] [user.real_name] (LV [user.get_text_level()]): Performed [work_type], [pe]/[max_boxes] PE."
+	SSlobotomy_corp.work_logs += "\[[worldtime2text()]\] [name]: [user_job_title] [user.real_name] (LV [user.get_text_level()]): Performed [work_type], [pe]/[max_boxes] PE."
```
Comment (LanceSmites328) [inline code/modules/mob/living/simple_animal/abnormality/waw/screenwriter.dm:106]: Note to self, find a way to make this unncessary

File: code/modules/mob/living/simple_animal/abnormality/waw/screenwriter.dm [hunk 2]
```diff
@@ -0,0 +1,376 @@
...
+
+/mob/living/simple_animal/hostile/actor/PickTarget(list/Targets)
+	var/list/priority = list()
+	for(var/mob/living/L in Targets)
+		if(!CanAttack(L))
+			continue
+		if(ishuman(L))
+			var/mob/living/carbon/human/H = L
+			if(H.sanity_lost) //ignore the panicked
+				if(H.ai_controller == /datum/ai_controller/insane/suicide/scene)
```
Comment (LanceSmites328) [inline code/modules/mob/living/simple_animal/abnormality/waw/screenwriter.dm:306]: Shouldn't this be a type check?

File: code/modules/mob/living/simple_animal/abnormality/waw/screenwriter.dm [hunk 3]
```diff
@@ -0,0 +1,376 @@
...
+					continue
+			else
+				priority += L
+		else
+			priority += L
+	if(LAZYLEN(priority))
+		return pick(priority)
+
+/mob/living/simple_animal/hostile/actor/AttackingTarget()
+	..()
```
Comment (LanceSmites328) [inline code/modules/mob/living/simple_animal/abnormality/waw/screenwriter.dm:316]:
```suggestion
	. = ..()
```

**Review by LanceSmites328 — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/simple_animal/abnormality/waw/screenwriter.dm [hunk 4]
```diff
@@ -0,0 +1,403 @@
...
+		"Cleanliness",
+		"Consensus",
+		"Amusement",
+		"Violence"
+						)
+
+//Init stuff
+/mob/living/simple_animal/hostile/abnormality/screenwriter/PostSpawn()
+	. = ..()
+	preferred_work_type = pick(work_types)
```
Comment (LanceSmites328) [inline code/modules/mob/living/simple_animal/abnormality/waw/screenwriter.dm:50]: Can't you do `pick(work_chances)`?

File: code/modules/mob/living/simple_animal/abnormality/waw/screenwriter.dm [hunk 5]
```diff
@@ -0,0 +1,403 @@
...
+	faction = list("hostile")
+	threat_level = WAW_LEVEL
+	start_qliphoth = 2
+	work_chances = list(
+		"Nutrition" = 35,
+		"Cleanliness" = 35,
+		"Consensus" = 35,
+		"Amusement" = 35,
+		"Violence" = 35
+	)
```
Comment (LanceSmites328) [inline code/modules/mob/living/simple_animal/abnormality/waw/screenwriter.dm:22]:
```suggestion
	)
	work_attribute_types = list(
		"Nutrition" = FORTITUDE_ATTRIBUTE,
		"Cleanliness" = PRUDENCE_ATTRIBUTE,
		"Consensus" = PRUDENCE_ATTRIBUTE,
		"Amusement" = TEMPERANCE_ATTRIBUTE,
		"Violence" = JUSTICE_ATTRIBUTE
	)
		
```
May need to rebase for this to work.

File: code/modules/mob/living/simple_animal/abnormality/waw/screenwriter.dm [hunk 6]
```diff
@@ -0,0 +1,403 @@
...
+
+	//Manually Add/Log PE
+	src.datum_reference.stored_boxes += pe
+	SSlobotomy_corp.AdjustAvailableBoxes(pe)
+	var/user_job_title = "Unidentified Employee"
+	var/obj/item/card/id/W = user.get_idcard()
+	if(istype(W))
+		user_job_title = W.assignment
+	datum_reference.work_logs += "\[[worldtime2text()]\] [name]: [user_job_title] [user.real_name] (LV [user.get_text_level()]): Performed [work_type], [pe]/[max_boxes] PE."
+	SSlobotomy_corp.work_logs += "\[[worldtime2text()]\] [name]: [user_job_title] [user.real_name] (LV [user.get_text_level()]): Performed [work_type], [pe]/[max_boxes] PE."
```
Comment (LanceSmites328) [inline code/modules/mob/living/simple_animal/abnormality/waw/screenwriter.dm:110]:
```suggestion
	preferred_work_type = pick(work_types)
	SpawnIcon()
```

**Review by LanceSmites328 — CHANGES_REQUESTED** [review]
Changes suggested are based on #1420

**Review by LanceSmites328 — APPROVED** [review]

**Pull request merged**